packages/venia-concept/src/components/CategoryList/categoryTile.js

```diff
@@ -12,12 +12,14 @@ const previewImageSize = 480;
 class CategoryTile extends Component {
     static propTypes = {
         item: shape({
-            image: string,
+            image: string.isRequired,
             name: string.isRequired,
             productImagePreview: shape({
                 items: arrayOf(
                     shape({
-                        small_image: string
+                        small_image: shape({
+                            url: string.isRequired
+                        })
```

Comment:
```suggestion
                        }).isRequired
```

Because you're accessing `small_image.url`, `render()` will throw if it's not an object.

Reply:
Added